lib/insert-link.js

```diff
@@ -101,9 +101,9 @@ function wrapClosestElement(node, dataAttrObject, matchValue) {
   }
 }
 
-function wrapsInnerString(text, matchValue, dataAttrObject) {
+export function wrapsInnerString(text, matchValue, dataAttrObject) {
```

Comment:
I would prefer if we keep this function private. Usually exposing functions just for the purpose of testing is a bad design which ends up most of the times in testing the implementation and not the behaviour. Ideally implementation changes should not affect tests unless the corresponding behaviour is changing as well. Therefor I prefer behaviour over implementation testing.